### ISSUE TITLE
Would you like some help clearing up the back PRs?

### ISSUE THREAD

**Issue by unixorn** [kahun/awesome-sysadmin#262]
@kahun, would you like some help getting these PRs triaged and merged?

**Comment by n1trux** [comment]
I think it's best if you merge the pull requests into your own repo and then create one big pull request.

Like so: https://gist.github.com/piscisaureus/3342247

**Comment by nodiscc** [comment]
@unixorn @kahun I'd like to help with this. There are currently 109 pull requests; I can merge them all in a seperate branch and propose merging this branch in a new PR. It's been almost a year with no updates. @kahun are you still interested in maintaining this repo?

**Comment by n1trux** [comment]
@nodiscc you could always consider contributing to http://www.github.com/n1trux/awesome-sysadmin :)

**Comment by nodiscc** [comment]
>  This branch is 243 commits ahead of kahun:master

Yep, I think I'll contribute to that :) Nice to see an active fork.

**Comment by ypid** [comment]
Hey guys

Nice to see that you are backing @kahun to maintain this project which really requires a lot of maintainable. Currently it seems to me that @kahun can not keep up with maintaining it alone and @n1trux is doing a good job in maintaining his fork.

To improve this situation a little future I am proposing that one of us creates an organization and @kahun moves https://github.com/kahun/awesome-sysadmin to this organization (since it is the most popular repo). When moving it, GitHub will redirect to the new URL (as long as @kahun does not create a new kahun/awesome-sysadmin after moving). Then @n1trux and others can be made contributors. This should help us all in the long run. Thoughts?

**Comment by unixorn** [comment]
@ypid I'm up for it. Merging @n1trux's fork into @kahun's after moving kahun/awesome-sysadmin to the new org seems like the best option.

@n1trux, you've been doing all the maintenance work, what would you prefer?

**Comment by n1trux** [comment]
I don't care if that repository is on my resume, so if you think that creating a new org for this will ensure continued maintenance, do it. I think just moving my repo into the new org would be easier, because else there'd be a small merging hassle. Is that a dumb idea?

There should be at least 2 or 3 people in charge of the new org, too, not only multiple contributors for the projects.

:+1: for awesome-foss. We could ask @awesome to rename himself for this – if he'd be so kind, we could have a nice, short URL (maybe /awesome/sysadmin?).

**Comment by ypid** [comment]
Thanks for your feedback.

> I don't care if that repository is on my resume

You can still do that with GitHub’ possibility to pin repositories on your profile page. Check: https://github.com/ypid

> because else there'd be a small merging hassle.

Not necessarily. We can still switch to your branch. I recommend that we move https://github.com/kahun/awesome-sysadmin to the new org as primary repository. @n1trux You can still keep your fork this way. After the new org has been set up, I think it would be save to start with your fork (you can open a PR and instead of merging, use something like `git checkout master-kahun && git reset --hard master-1trux`.) This way, all people who know https://github.com/kahun/awesome-sysadmin would be redirected to the new org. Doing it your way would mean that we would need to add a description to https://github.com/kahun/awesome-sysadmin saying something like "unmaintained, please go to here".

> There should be at least 2 or 3 people in charge of the new org, too, not only multiple contributors for the projects.

We are at least four people who care enough about this to write a comment here :wink: So I hope that works out.

>  /awesome/sysadmin

That would be awesome :smile: but I am not sure if @awesome would do that. Not sure if I would. I think if we don’t here from Mr. @awesome we can go with awesome-foss.

@n1trux As you already have experience with maintaining this project (and your related project [awesome-donations](https://github.com/n1trux/awesome-donations) which I would suggest to also move then) would you like to create the org? I could also do it if you want but I am not as much involved with this project as you are.

**Comment by n1trux** [comment]
Yeah, I'll move awesome-donations (and awesome-beers). Let's wait on feedback from @kahun and @awesome :)

**Comment by unixorn** [comment]
I'm willing to be a contributor. I'm a contributor for [dotfiles.github.com](https://github.com/dotfiles/dotfiles.github.com) and am the primary author for [awesome-zsh-plugins](https://github.com/unixorn/awesome-zsh-plugins).

**Comment by ypid** [comment]
Someone active on g+ or knows enough Spanish to find another way to contact @kahun?

**Comment by irabinovitch** [comment]
Happy to reach out to him in Spanish if we think it'd help. Although the issue is likely not language and more just that like everyone his github notifications are super noisy/spammy.
